src/main/java/teammates/ui/template/CourseTableSessionRow.java

```diff
@@ -0,0 +1,38 @@
+package teammates.ui.template;
+
+public class CourseTableSessionRow {
+    private String name;
+    private String tooltip;
+    private String status;
+    private String href;
+    private String recent;
+    private String actions;
+    
+    public CourseTableSessionRow(String name, String tooltip, String status, String href, String recent, String actions) {
```

Comment:
Add a header comment for this method. Some of the variable names are not self-explanatory.